bin/cherry

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/bin/bash
```

Comment:
Don't think this is correct. Below you are using Python syntax to import a module; so this file has to be executed by the Python interpreter, no?

Or was the idea to switch to bash but then do something like

    python3 python3 -m cherry.cli "$@"

?

https://github.com/moabitcoin/ig65m-pytorch/blob/fc749e2ee354c3e4ddbb144cf511bb868b008f61/bin/ig65m